site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -175,7 +181,11 @@ export type CreateTemplateFormProps = (
 };
 
 export const CreateTemplateForm: FC<CreateTemplateFormProps> = (props) => {
+  const { experiments } = useDashboard();
+  const { multiple_organizations: organizationsEnabled } =
+    useFeatureVisibility();
   const [searchParams] = useSearchParams();
+  const [selectedOrg, setSelectedOrg] = useState<Organization | null>(null);
```

Comment:
I noticed that you use `null` here while most of the codebase uses `undefined`. Is there a specific reason for that?

Reply:
because MUI uses `null` when clearing the `Autocomplete` component backing this org picker